SonarRunner.Shim/PropertiesWriter.cs

```diff
@@ -200,9 +218,12 @@ private static string ComputeProjectBaseDir(IEnumerable<ProjectInfo> projects, s
             try
             {
                 var commonParts = new List<string>();
-                while (pathPartEnumerators.Any() && pathPartEnumerators.All(e => e.MoveNext()) && pathPartEnumerators.All(e => e.Current == pathPartEnumerators.First().Current))
+                if (pathPartEnumerators.Length > 0)
```

Comment:
Fix for unrelated bug that was causing unit test failures: guard against an empty list.